operations/template/alert.tf

```diff
@@ -102,8 +103,49 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "database_token_expired_a
     threshold = 1
   }
 
-  #   below tags are managed by CDC
   lifecycle {
+    # Ignore changes to tags because the CDC sets these automagically
+    ignore_changes = [
+      tags["business_steward"],
+      tags["center"],
+      tags["environment"],
+      tags["escid"],
+      tags["funding_source"],
+      tags["pii_data"],
+      tags["security_compliance"],
+      tags["security_steward"],
+      tags["support_group"],
+      tags["system"],
+      tags["technical_steward"],
+      tags["zone"]
+    ]
+  }
+}
+
+resource "azurerm_monitor_metric_alert" "azure_4XX_alert" {
+  count               = local.non_pr_environment ? 1 : 0
+  name                = "cdcti-${var.environment}-azure-http-4XX-alert"
+  resource_group_name = data.azurerm_resource_group.group.name
+  scopes              = [data.azurerm_resource_group.group.id]
```

Comment:
We don't want the entire resource group checked, right?  Since this is in the TI repo, we want this specific to TI, so should this be perhaps `azurerm_linux_web_app.api.id`?

Reply:
@saquino0827, I think all I'm waiting for is addressing this feedback (or having a conversation around this if you feel this isn't right).  After that, I'm ready to approve.

Reply:
Oh, and also probably want to make sure this actually works in some environment.

Reply:
@halprin this has been updated. Currently testing in internal and waiting for the alert that was fired to reach resolved status